doc/programming_model/attributes_quantization.md

```diff
@@ -73,23 +73,32 @@ time.
 
 ### Dynamic Quantization
 
-The only formula for dynamic quantization currently supported by
-oneDNN is with scales computed following the
-[OCP MX Formats Specification][mx-spec], namely:
+
+oneDNN supports two dynamic quantization modes, for scales only (no
+zero-point) following the formula:
 
 \f[
 x_{f32}[:] = scale_{x} \cdot x_{quant}[:]
 \f]
 
 where \f$x_{f32}\f$ and \f$x_{quant}\f$ are the non-quantized and
 quantized representation of \f$x\f$ respectively, and \f$scale_{x}\f$ is a
-scaling factor:
-- in `e8m0` format,
-- computed for each group of size `32`,
-- and computed as the largest power-of-two less than or equal to the
-  maximum absolute value of the group divided by the largest
-  power-of-two representable in the \f$x_{quant}\f$ data type,
-  e.g., \f$E8M0(amax(x_{quant}[:])) / E8M0(MAX\_QUANT\_DT) \f$.
+scaling factor.
+
+When using `quantization_mode::dynamic_mx`, \f$scale_{x}\f$ is computed
+following the [OCP MX Formats Specification][mx-spec], namely \f$scale_{x}\f$:
+- has `e8m0` datatype,
+- is computed for each group of size `32`,
+- is computed as the largest power-of-two less than or equal to the maximum
+  absolute value of the group divided by the largest power-of-two representable
+  in the \f$x_{quant}\f$ datatype, e.g.,
+  \f$E8M0(amax(x_{quant}[:])) / E8M0(MAX\_QUANT\_DT)\f$.
+
+When using `quantization_mode::dynamic_fp`, \f$scale_{x}\f$ is computed in
```

Comment:
Similarly to mx, could we reference some sort of spec here with full details wrt data types?

Reply:
Not that I'm aware of. Once it becomes available, it'll be added.